Resolver/DoctrineObjectEventResolver.php

```diff
@@ -32,9 +31,9 @@ class DoctrineObjectEventResolver implements EventResolverInterface
     protected $identity = ['', ''];
 
     /**
-     * @var ManagerRegistry
+     * @var \Doctrine\ORM\EntityManager
```

Comment:
As we also support ODM,  we don't want direct dependency with ORM. You can see we remove the dependency in a earlier [commit](https://github.com/xiidea/EasyAuditBundle/commit/648baeaf89f02be8e872fa523f94756bbc6f4cfd#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L27)